assets/js/modules/analytics-4/components/dashboard/ActivationBanner/SetupBanner.js

```diff
@@ -19,25 +19,91 @@
 /**
  * WordPress dependencies
  */
-import { __ } from '@wordpress/i18n';
+import { __, sprintf } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
+import Data from 'googlesitekit-data';
 import BannerNotification from '../../../../../components/notifications/BannerNotification';
+import { Grid, Row, Cell } from '../../../../../material-components';
+import { MODULES_ANALYTICS_4 } from '../../../datastore/constants';
+const { useSelect } = Data;
 
 export default function SetupBanner( { onCTAClick } ) {
+	const ga4MeasurementID = useSelect( ( select ) =>
+		select( MODULES_ANALYTICS_4 ).getMeasurementID()
+	);
+	const existingTag = useSelect( ( select ) =>
+		select( MODULES_ANALYTICS_4 ).getExistingTag()
+	);
+
+	let title;
+	let ctaLabel;
+	let footer;
+
+	if ( ! ga4MeasurementID && ! existingTag ) {
+		title = __(
+			'No existing Google Analytics 4 property found, Site Kit will help you create a new one and insert it on your site',
+			'google-site-kit'
+		);
+		ctaLabel = __( 'Create property', 'google-site-kit' );
+		footer = __(
+			'You can always add/edit this in the Site Kit Settings.',
+			'google-site-kit'
+		);
+	} else if ( ga4MeasurementID && ! existingTag ) {
+		title = __(
+			'Connect the Google Analytics 4 property that’s associated with your existing Universal Analytics property',
+			'google-site-kit'
+		);
+		ctaLabel = __( 'Connect', 'google-site-kit' );
+		footer = __(
+			'You can always add/edit this in the Site Kit Settings.',
+			'google-site-kit'
+		);
+	} else if ( ga4MeasurementID && existingTag ) {
+		title = __(
+			'Connect the Google Analytics 4 property that’s associated with your existing Universal Analytics property',
+			'google-site-kit'
+		);
+		ctaLabel = __( 'Connect', 'google-site-kit' );
+		footer = __(
+			'You can always add/edit this in the Site Kit Settings.',
+			'google-site-kit'
+		);
```

Comment:
These two cases are identical, so we can combine them. I suggest using the following approach:


```js
if ( ga4MeasurementID ) {
    title = __( 'Connect the Google Analytics 4 property that’s associated with ...', ... );
    ctaLabel = __( ... );
    footer = __( ... );
} else if ( existingTag ) {
    title = __( ... );
    ctaLabel = __( ... );
    footer = __( 'A GA4 tag %s is found on this site but ...', ... );
} else {
    title = __( 'No existing Google Analytics 4 property found, ...', ... );
    ctaLabel = __( ... );
    footer = __( ... );
}
```